frb_codegen/src/library/codegen/generator/misc/mod.rs

```diff
@@ -33,10 +33,14 @@ pub fn is_js_value(ty: &MirType) -> bool {
     }
 }
 
-pub(crate) fn generate_code_header() -> String {
-    format!(
-        "// This file is automatically generated, so please do not edit it.
-// @generated by `flutter_rust_bridge`@ {}.",
-        env!("CARGO_PKG_VERSION")
-    )
+#[macro_export]
+macro_rules! generate_code_header {
```

Comment:
nit: would be great to be a function (that may return `&'static str` if you like) to avoid pitfalls of macros (such as too public)

Reply:
I made `generate_code_header` a macro as `concat!()` can't accept a function returning `&'static str`.

Reply:
Totally agree, but as is discussed in other PRs you created, the PR looks like premature optimization :( So maybe we can keep those that decrease readability little.